qutebrowser/browser/webengine/interceptor.py

```diff
@@ -85,6 +87,20 @@ def install(self, profile):
         """Install the interceptor on the given QWebEngineProfile."""
         profile.setRequestInterceptor(self)
 
+    # pylint: disable=bad-builtin
+    WHITELIST_REQUEST_METHODS = frozenset(map(QByteArray, [b'GET', b'HEAD']))
```

Comment:
I don't think we are allowed to use `map`. There is only one instance of it in the codebase that I can find and that is from a while ago 1570fc8a.
Also should this be declared in amongst the method declarations?

Reply:
Why not just `{QByteArray(b'GET'), QByteArray(b'HEAD')}` though? Much more readable IMHO, shorter, and it's not like we're going to have thousands of request methods here.